src/components/Monster/Monster.jsx

```diff
@@ -1,21 +1,33 @@
 import React from 'react';
-import style from './Monster.module.css';
+import { connect } from 'react-redux';
+import styles from './Monster.module.css';
+import classNames from 'classnames/bind';
+
+const cx = classNames.bind(styles);
```

Comment:
I've never seen `classnames.bind` before, seems interesting. I _think_ I can see what it's doing.

Reply:
to me it just looks like not having to write className1 + " " + className 2 :D

Reply:
Yeah but `classnames` does that without having to use `.bind`.

Reply:
I saw it online as when you try to do `classNames({ styles.class: condition })` it doesn't like `styles.class` so you have to bind styles to it before?